tracer/src/Datadog.Trace/Processors/TraceUtil.cs

```diff
@@ -57,6 +57,11 @@ public static bool IsValidStatusCode(string statusCode)
         // https://github.com/DataDog/datadog-agent/blob/eac2327c5574da7f225f9ef0f89eaeb05ed10382/pkg/trace/traceutil/normalize.go#L98-L209
         public static string NormalizeTag(string value)
         {
+            if (value is null)
+            {
+                return null;
+            }
```

Comment:
This is technically distinct from the issue described right? It makes sense, but wonder if we should add an explicit test to `NormalizerTraceProcessorTests` (it should probably have already been testing the `null` case?)

Reply:
Well is related, because a tag in span will have prevalence over the payload, even if is empty string. I’ll check the test.

Reply:
Test fixed in [4c5d613](https://github.com/DataDog/dd-trace-dotnet/pull/2783/commits/4c5d613eaa513af945d6df3c1584e4c6b4c67f1f)